lisa/wlgen/rta.py

```diff
@@ -553,7 +553,8 @@ class Phase(Loggable):
     :type barrier_after: str
     """
 
-    def __init__(self, duration_s, period_ms, duty_cycle_pct, cpus=None, barrier_after=None):
+    def __init__(self, duration_s, period_ms, duty_cycle_pct, cpus=None, barrier_after=None,
+                 uclamp_min=None, uclamp_max=None):
```

Comment:
While packing them into a single tuple might make it look tidier, I think we want to keep this interface. AFAICT there is room for more uclamp settings, so this is a more future-proof API (that or creating a uclamp class to pass in as parameter...).